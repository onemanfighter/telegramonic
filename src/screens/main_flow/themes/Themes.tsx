const Themes = () => {
  return (
    <div>
      <h1>Themes</h1>
      <p>Themes page content</p>
    </div>
  );
};

export default Themes;
