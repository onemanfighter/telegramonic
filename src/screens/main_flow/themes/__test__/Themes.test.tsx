import { render } from '@testing-library/react';
import Themes from '../Themes';

describe('Themes screen', () => {
  it('should render correctly', () => {
    const { container } = render(<Themes />);

    expect(container).toMatchSnapshot();
  });
});
