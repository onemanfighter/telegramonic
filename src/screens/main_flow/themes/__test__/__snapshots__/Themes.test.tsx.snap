// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Themes screen should render correctly 1`] = `
<div>
  <div>
    <h1>
      Themes
    </h1>
    <p>
      Themes page content
    </p>
  </div>
</div>
`;
